cmd/stdiscosrv/main.go

```diff
@@ -99,10 +99,12 @@ func main() {
 
 	cert, err := tls.LoadX509KeyPair(certFile, keyFile)
 	if err != nil {
-		log.Println("Failed to load keypair. Generating one, this might take a while...")
-		cert, err = tlsutil.NewCertificate(certFile, keyFile, "stdiscosrv", 20*365)
-		if err != nil {
-			log.Fatalln("Failed to generate X509 key pair:", err)
+		if os.IsNotExist(err) {
```

Comment:
We still need to handle the case when there is an error but it's not "doesn't exist". I suggest 

```
if os.IsNotExist(err) {
  ... what you did
} else if err != nil {
  log.Fatalln("Failed to load keypair:", err
}
```